openapi/spec/openapi.json

```diff
@@ -2692,6 +2745,17 @@
         },
         "required": ["title", "recipient"]
       },
+      "Delivery": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["id", "channel", "scheduled_at", "status"],
+        "properties": {
+          "id": { "type": "string" },
+          "channel": { "type": "string" },
+          "scheduled_at": { "type": "string", "format": "date-time" },
+          "status": { "type": "integer", "description": "0: unseen, 5: unread, 10: read, 20: archived" }
```

Comment:
I've changed this to a string enum. Supported values as of now are `processing`, `invalid`, `scheduled`, and `send`.

Reply:
I've also added a `Title` (string) property which makes it easy to check the title of the delivery - something that can be customized using template, or overrides.